src/main/scala/lisa/kernel/fol/FormulaLabelDefinitions.scala

```diff
@@ -41,12 +41,12 @@ private[fol] trait FormulaLabelDefinitions extends CommonDefinitions {
   /**
    * A standard predicate symbol. Typical example are equality (=) and membership (∈)
    */
-  final case class ConstantPredicateLabel(id: String, arity: Int) extends PredicateLabel
+  final case class ConstantPredicateLabel(id: String, arity: Int) extends PredicateLabel with TheorySymbol
 
   /**
    * The equality symbol (=) for first order logic.
    */
-  val equality: PredicateLabel = ConstantPredicateLabel("=", 2)
+  val equality: ConstantPredicateLabel = ConstantPredicateLabel("=", 2)
```

Comment:
:+1:
Solves the [ugly cast](https://github.com/FlorianCassayre/master-project/blob/master/src/main/scala/me/cassayre/florian/masterproject/front/theory/SetTheory.scala#L36) I had to do.